cmd/evm/internal/t8ntool/transition.go

```diff
@@ -390,21 +395,21 @@ func VerkleKey(ctx *cli.Context) error {
 		return fmt.Errorf("error decoding address: %w", err)
 	}
 
-	ap := utils.EvaluateAddressPoint(addr)
 	if ctx.Args().Len() == 2 {
 		slot, err := hexutil.Decode(ctx.Args().Get(1))
 		if err != nil {
 			return fmt.Errorf("error decoding slot: %w", err)
 		}
-		fmt.Printf("%#x\n", utils.GetTreeKeyStorageSlotWithEvaluatedAddress(ap, slot))
+		fmt.Printf("%#x\n", bintrie.GetBinaryTreeKeyStorageSlot(common.BytesToAddress(addr), slot))
 	} else {
-		fmt.Printf("%#x\n", utils.GetTreeKeyBasicDataEvaluatedAddress(ap))
+		var zeroSlot [32]byte
+		fmt.Printf("%#x\n", bintrie.GetBinaryTreeKeyStorageSlot(common.BytesToAddress(addr), zeroSlot[:]))
```

Comment:
please implement `GetBinaryTreeKeyBasicData` which is pretty much the same as `GetBinaryTreeKeyCodeHash` except the offset is 0 instead of 1